CODEOWNERS

```diff
@@ -49,8 +49,8 @@
 /builder/proxmox/                      @carlpett
 /website/pages/docs/builders/proxmox* @carlpett
 
-/builder/scaleway/                      @sieben @mvaude @jqueuniet @fflorens @brmzkw
-/website/pages/docs/builders/scaleway* @sieben @mvaude @jqueuniet @fflorens @brmzkw
+/builder/scaleway/                      @scaleway/devtools
+/website/pages/docs/builders/scaleway* @scaleway/devtools
```

Comment:
ah, I didn't know we could do that this is awesome !!

Reply:
It sure is :) https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners